proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+# Cosmos Stargate Hub Upgrade Proposal 2: Chevrons Locked. Time to Upgrade
+
+![Stargate Upgrade banner graphic](https://siasky.net/AADU5rg0GvapbrICLUk7SxZeMDrKo5Q8oLrpOwl71wXhnQ)
+
+## by Iqlusion
+
+Jan 11, 2021
+
+## Key Results
+
+------------
+If passed, this governance proposal would commit the Cosmos Hub to halting the `cosmoshub-3` at 06:00 UTC on Jan 28th exporting the state and starting `cosmoshub-4` based on gaia 3.0.
```

Comment:
I want to push for whatever date that is put here is at least 36 hours after expected proposal accept / reject time, using average block time. 

48 hours because variance in expected proposal accept/reject time over 2 weeks could easily alter +- 12 hours, and I think a minimum notice of 24 hours to see the final decision before needing to act upon it is reasonable.

Reply:
How long does voting usually take?

Reply:
@tessr voting period is 2 weeks, same as deposit period

Reply:
Yes strong agree with min 36 hours window.